apps/accounts/views/mfa.py

```diff
@@ -88,6 +90,17 @@ def mfa_login(request):
         if form.is_valid():
             username = form.cleaned_data['username']
             password = form.cleaned_data['password']
+
+            # If username doesn't exist, try username matching email address.
+            try:
+                User.objects.get(username__iexact=username)
+            except User.DoesNotExist:
+                try:
+                    check_user = User.objects.get(email__iexact=username)
+                    username = check_user.username
+                except User.DoesNotExist:
+                    pass
```

Comment:
I think this should still raise, although `authenticate` will probably take care of that too.

Reply:
yup nm `authenticate` would already have been handling this, no reason to change that behavior